modules/project-docs/pages/sdk-release-notes.adoc

```diff
@@ -28,11 +28,68 @@ Java 11 has various enhancements like lambda local variable type inference, prof
     <dependency>
         <groupId>com.couchbase.client</groupId>
         <artifactId>java-client</artifactId>
-        <version>3.0.4</version>
+        <version>3.0.5</version>
     </dependency>
 </dependencies>
 ----
 
+== Version 3.0.5 (2 June 2020)
+
+Version 3.0.5 is the sixth release of the 3.0 series, bringing enhancements and bugfixes over the last stable release.
+
+http://packages.couchbase.com/clients/java/3.0.5/Couchbase-Java-Client-3.0.5.zip[Download] |
+http://docs.couchbase.com/sdk-api/couchbase-java-client-3.0.5/[API Reference] |
+http://docs.couchbase.com/sdk-api/couchbase-core-io-2.0.6/[Core API Reference]
+
+The supported and tested dependencies for this release are:
+
+* com.couchbase.client:**java-client:3.0.5**
+* com.couchbase.client:**core-io:2.0.6**
+* io.projectreactor:**reactor-core:3.3.5.RELEASE**
+* org.reactivestreams:**reactive-streams:1.0.3**
+
+=== Enhancements
```

Comment:
It seems that some tickets are not mentioned, like : JVMCBC-693 enhancement, bugfix JVMCBC-849 and volatile Field-Level Encryption support (note that it will have to be used with the enterprise only extension)

Reply:
I don't know how I'm meant to discover these though.

JVMCBC-693 is filed for 2.1.0 (and it's In Progress)
849 is marked as In Progress.

I'm going by the Done tabs on these two releases
https://issues.couchbase.com/projects/JVMCBC/versions/16780
https://issues.couchbase.com/projects/JCBC/versions/16781